src/transformers/models/gemma2/configuration_gemma2.py

```diff
@@ -98,7 +99,6 @@ class Gemma2Config(PretrainedConfig):
 
     model_type = "gemma2"
     keys_to_ignore_at_inference = ["past_key_values"]
-    cache_implementation = "hybrid"
```

Comment:
TIL: attributes defined at a class level don't are not present in `__dict__`. Not present in `__dict__` -> `config.to_dict()` doesn't contain `cache_implementation` -> `generation_config` is not initialized with `cache_implementation` -> tests fail because they expect `HybridCache`